css/components/input.css

```diff
@@ -0,0 +1,39 @@
+/* 
+  Input
+*/
+
+.input {
```

Comment:
Just a suggestion using input as a class might result in confusion at times. better practice would be to avoid naming with variables that have html tags.

Reply:
Thanks for the suggestion @Nishant-96 , Switched the name to input-field